speech/cloud-client/beta_snippets.py

```diff
@@ -126,6 +127,35 @@ def transcribe_file_with_auto_punctuation(path):
 # [END speech_transcribe_file_with_auto_punctuation]
 
 
+# [START speech_transcribe_multilanguage]
+def transcribe_file_with_multilanguage(speech_file):
+    """Transcribe the given audio file synchronously with
+      multi language."""
+    from google.cloud import speech_v1p1beta1 as speech
+    client = speech.SpeechClient()
+
+    with open(speech_file, 'rb') as audio_file:
+        content = audio_file.read()
+
+    audio = speech.types.RecognitionAudio(content=content)
+
+    config = speech.types.RecognitionConfig(
+        encoding=speech.enums.RecognitionConfig.AudioEncoding.LINEAR16,
+        sample_rate_hertz=16000,
+        language_code='ja-JP',
+        alternative_language_codes=['es-ES', 'en-US'])
```

Comment:
How does this work?  if the speech in the audio has multiple languages in it, say English and Japanese, then the transcription will have both?